scripts/validate_docstrings.py

```diff
@@ -471,6 +471,36 @@ def parameter_desc(self, param):
                 desc = desc[:desc.index(full_directive)]
         return desc
 
+    def parameter_desc_list(self, param):
+        i = list(self.doc_parameters).index(param)
+        desc_list = [line
+                     for line in self.doc._parsed_data['Parameters'][i][-1]
+                     if line]
+        # Find and strip out any sphinx directives
+        for directive in DIRECTIVES:
+            full_directive = '.. {}'.format(directive)
+            for desc_item in desc_list:
+                if full_directive in desc_item:
+                    # Only retain any description before the directive
+                    desc_list = desc_list[:desc_list.index(desc_item)]
+                    break
+        return desc_list
+
+    def parameter_correct_end(self, param):
+        if self.parameter_desc(param)[-1] == '.':
+            return True
+        else:
+            # Return True if a parameter description ends
+            # with a bullet point
+            for desc_line in self.parameter_desc_list(param)[::-1]:
+                if desc_line[0] in ['*', '-', '+']:
```

Comment:
Note that there are quite a few more markers technically to be accounted for:

http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#bullet-lists

Reply:
Thank you for this, I did not see the extra ones before.